src/main/resources/main_sv.properties

```diff
@@ -0,0 +1,48 @@
+# Main text translations
+main.startPrompt=Kommandon: uppskatta l\u00F6senord (e); generera l\u00F6senord (g); sluta (q)
+main.enterCommand=Var god och mata in ditt kommando:
+main.estimatePrompt=Var god och mata in l\u00F6senordet du vill uppskatta:
+main.generatorType=Vilken typs av l\u00F6senord vill du generera? Slumpm\u00E4ssig (r), L\u00F6senfras (p):
+main.randomType=Alfa (1), Alfanumerisk (2), Alfanumerisk & Symbol (3), Numerisk (4):
+main.lengthPrompt=Hur m\u00E5nga tecken:
+main.delimiterPrompt=Vilken avgr\u00E4nsare vill du anv\u00E4nda:
+main.wordsPrompt=Hur m\u00E5nga ord vill du att ditt l\u00F6senfras inneh\u00E5ller:
+main.quitPrompt=Tack f\u00F6r att du anv\u00E4nder nbvcxz!
+main.timeToCalculate=Tid att r\u00E4kna:
+main.password=L\u00F6senord:
+main.feedback.warning=Varning:
+main.feedback.suggestion=F\u00F6rslag:
+main.feedback.minimumMet=Ditt l\u00F6senord uppfyller minimi styrke kraven.
+main.feedback.insecure=Ditt l\u00F6senord uppfyller inte minimi styrke kraven.
+main.entropy=Entropi:
+main.timeToCrack=Tid att kn\u00E4cka:
+main.match.matchType=Typ av match:
+main.match.token=Token:
+main.match.startIndex=Startindex:
+main.match.endIndex=Slutindex:
+main.match.length=L\u00E4ngd:
+main.match.dictionary=Ordbok:
+main.match.dictionaryValue=Ordboksv\u00E4rde:
+main.match.rank=Rang:
+main.match.leetSub=Leet Ers\u00E4ttningar:
+main.match.reversed=Omv\u00E4nd:
+main.match.distance=Avst\u00E5nd:
+main.match.repeatingCharacters=Upprepade tecknena:
+main.match.repeated=Upprepat:
+main.match.firstCharacter=F\u00F6rsta tecknet:
+main.match.spacialType=Tangentbordets typ:
```

Comment:
Same as in Finnish, restricting spatial types to keyboard types shouldn't probably be done. Something like "Spatialtyp:" perhaps?

Reply:
amended